controller/controller_server/utils.py

```diff
@@ -154,7 +274,7 @@ def validate_delete_snapshot_request(request):
         raise ValidationException(messages.name_should_not_be_empty_message)
     logger.debug("validating secrets")
     if request.secrets:
-        validate_secret(request.secrets)
+        validate_secrets(request.secrets)
```

Comment:
is this `if` still needed? if not, could you remove it while you're at it?

Reply:
done